packages/metrics/src/features/metrics/utils.ts

```diff
@@ -146,11 +150,44 @@ export const fetchEstimateByLocation = async (
     if (!locationData.partOf || !locationData.partOf.reference) {
       throw new Error('Unable to fetch estimate data from location tree')
     }
-    return await fetchEstimateByLocation(locationData, estimatedYear)
+    return await fetchEstimateByLocation(
+      await fetchFHIR(locationData.partOf.reference, authHeader),
+      estimatedYear,
+      authHeader
+    )
   }
   return {
     estimation: Math.round((crudRate * population) / 1000),
-    locationId: locationData.id
+    locationId: locationData.id,
+    estimationYear: actualEstimationYear,
+    locationLevel: getLocationLevelFromLocationData(locationData)
+  }
+}
+
+export const getLocationLevelFromLocationData = (locationData: Location) => {
+  return (
+    locationData?.identifier?.find(
+      identifier =>
+        identifier.system ===
+        OPENCRVS_SPECIFICATION_URL + JURISDICTION_TYPE_IDENTIFIER
+    )?.value ?? ''
+  )
+}
+
+export const fetchEstimateFor45DaysByLocationId = async (
+  locationId: string,
+  estimatedYear: number,
+  authHeader: IAuthHeader
+): Promise<IEstimation> => {
+  const locationData: Location = await fetchFHIR(locationId, authHeader)
+  const estimate = await fetchEstimateByLocation(
+    locationData,
+    estimatedYear,
+    authHeader
+  )
+  return {
+    ...estimate,
+    estimation: Math.round(estimate.estimation / 8.111111) // for 45 day
```

Comment:
Sorry, the required calculation has changed.  Please check: https://jembiprojects.jira.com/browse/OCRVS-2761